### ISSUE TITLE
Instant crash if loading file after any kind of interaction with scene or viewports

### ISSUE THREAD

**Issue by cesm1980** [PredatorCZ/HavokMax#4]
I am using version 1.6 with max 2012 64bit. Well, this one is quite unsual, if i do any kind of interaction with the scene ou any of the viewports, as soon as i try to load a file with your plugin, it makes max crash! And there's more, since the plugin is always in deferred mode when i start max (except for when using it the first time), if i do any changes i want on the viewport at all and only after that i manually load the plugin, it doesn't crash anymore, so it's only the interactions with the viewport AFTER the plugin is loaded that creates the problem.

So far the things i tested was : maximizing/restoring viewport (even using just keyboard has the same problem so it's not related to mouse clicking), clicking anywhere on the scene, clicking on the toolbar buttons to restrict to a single axis (yes even this does trigger the problem even without touching the viewport) etc etc, it's VERY easy to trigger this in fact...

**Comment by PredatorCZ** [comment]
Does this happen with other importers?
Does it happens for other versions as-well?
Do bones show in the viewports at all?
Are you importing in deferred mode?

**Comment by PredatorCZ** [comment]
Please try V1.8, if that fixed something.

**Comment by cesm1980** [comment]
Like i said on the other issue, i tried version 1.8 and wow it's a major improvement, i tried everything i could to crash the plugin and wasn't able to :D I even loaded in sucession the skeleton file, then a animation and then again the skeleton file since this would very frequently also trigger the crash, now EVEN new scene works perfectly too and doesn't make it crash, even when using in deferred mode (since max auto-loads it now), congratulations, to me this was one of the most serious bugs it had, even tough i am not sure if this happened in the max version you had, but it's definetly fixed now.

Oh there's two other bug fixex i found yesterday that i haven't told you yet that got fixed now : 
-loading files with special characters like ç á ã would immediatly make the plugin crash, unintentionally or not you fixed this too!
-in version 1.6 and previous ones, when bones were missing the maxscript window would only say "warning warning warning" and now it displays the complete bone names and even auto-pops the maxscript window, very useful!

**Comment by PredatorCZ** [comment]
Versions 2010 - 2012 are not using unicode, I would be very careful when loading non ascii characters.
Codepages are nightmare to me, since dumb windows doesn't support utf-8 at all, it will accept only characters for your local codepage, when using non-unicode apps.

**Comment by cesm1980** [comment]
Yeah, altough something changed in version 1.8 (or 1.7 but i haven't tested that one) that fixed this, not sure why, also i never had problems importing files with those characters using other file importer plugins so it was something else then.

To me this issue seems solved for now, if i find another crash trigger i re-open this.
